google/cloud/spanner/samples/samples.cc

```diff
@@ -4887,6 +4930,9 @@ void RunAll(bool emulator) {
   SampleBanner("spanner_set_request_tag");
   SetRequestTag(client);
 
+  SampleBanner("spanner_directed_read");
```

Comment:
nit: at 5000+ lines, maybe it is time to split this file?   It requires some planning because one cannot just remove region tags from a file...

Reply:
Yeah.  I'm in two minds.  This really is a tangle, with each sample largely dependent on the state left by those before it.  So, any splitting doesn't really make it easier to understand.  But I'll give some more thought to a better state, and how we might get there.